boltdesignsystem/bolt
# DS-520 Refactor Page Footer component

## Jira

https://pegadigitalit.atlassian.net/browse/DS-520

## Summary

Refactored Page Footer component.

## Details

1. HTML was reduced by removing bands, wrappers, and grid.
2. Layout is responsive across all breakpoints.
3. All texts and links have at least AA color contrast.
4. Proper semantics and aria attributes are added.
5. Proper interactive states (hover, active, focus, etc.) are added for all links.
6. Mobile design has been enhanced.

## How to test

Run the branch locally and check if Page Footer component conforms to accessibility standards listed in Details.
